lib/Doctrine/ORM/Mapping/ReflectionEnumProperty.php

```diff
@@ -56,6 +58,12 @@ public function getValue($object = null)
             return null;
         }
 
+        if (is_array($enum)) {
+            return array_map(static function ($item): mixed {
```

Comment:
```suggestion
            return array_map(static function (BackedEnum $item): mixed {
```

Reply:
Changed in [b6e7472](https://github.com/doctrine/orm/pull/9497/commits/b6e7472f91ffe29851eb08093f6f8c1e8eb817a8)